modules/generative-aws/config/class_settings.go

```diff
@@ -0,0 +1,385 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/entities/moduletools"
+)
+
+const (
+	serviceProperty           = "service"
+	regionProperty            = "region"
+	modelProperty             = "model"
+	endpointProperty          = "endpoint"
+	targetModelProperty       = "targetModel"
+	targetVariantProperty     = "targetVariant"
+	maxTokenCountProperty     = "maxTokenCount"
+	maxTokensToSampleProperty = "maxTokensToSample"
+	stopSequencesProperty     = "stopSequences"
+	temperatureProperty       = "temperature"
+	topPProperty              = "topP"
+	topKProperty              = "topK"
+)
+
+var (
+	DefaultTitanMaxTokens     = 8192
+	DefaultTitanStopSequences = []string{}
+	DefaultTitanTemperature   = 0.0
+	DefaultTitanTopP          = 1.0
+	DefaultService            = "bedrock"
+)
+
+var (
+	DefaultAnthropicMaxTokensToSample = 300
+	DefaultAnthropicStopSequences     = []string{"\\n\\nHuman:"}
+	DefaultAnthropicTemperature       = 1.0
+	DefaultAnthropicTopK              = 250
+	DefaultAnthropicTopP              = 0.999
+)
+
+var DefaultAI21MaxTokens = 300
+
+// DefaultAnthropicStopSequences = []string{"\\n\\nHuman:"}
+// DefaultAnthropicTemperature   = 1.0
+// DefaultAnthropicTopK          = 250
+// DefaultAnthropicTopP          = 0.999
```

Comment:
Still needed?

Reply:
I have removed those settings I don't thing that it's needed. When I'll be re-writing this module I'll then re-introduce them if they will be still needed